### PR TITLE
6.2.6 release notes

### PR DESCRIPTION
**Pull request by Chr1st0ph3rTurn3r**
Waiting on text for WAN 3027

### CODE REVIEW

**Comment by agrawalkaushik** [comment]
@Chr1st0ph3rTurn3r 

> Waiting on text for WAN 3027

This is a WAN Assurance JIRA and not applicable to 6.2.6.

File: docs/release_notes_128t_6.2.md [hunk 1]
```diff
@@ -24,6 +24,110 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+------
+- **I95-49712 Config Validation Error uniformative:** Resolved an issue where paths pointing to missing keys or elements returned error messages that did not provide recognizable information. Now the error message provides key or element names.
+------ 
+- **I95-55725 Highway crashes when peer-path routers are removed:** Resolved a race condition that could cause a crash in the highway worker-core packet-processor if peer routers are removed from the configuration.
+------
+- **I95-55965 IDP engine not starting due to invalid environmental conditions:** In cases where a container/csrx does not shut down cleanly, the IDP engine does not start. These conditions are now detected, and a force stop/remove of the container is implemented.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.2.md:51]: Too many internal details:

- **I95-55965 IDP engine not starting due to invalid environmental conditions:** In cases where the IDP engine does not shut down cleanly, the IDP engine will fail to restart. These conditions are now detected and handled correctly.

File: docs/release_notes_128t_6.2.md [hunk 2]
```diff
@@ -24,6 +24,110 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+------
+- **I95-55982 X722 interfaces overwritten when upgrading an SSR1300/SSR1400:** Identified an issue where the firmware for the x722 interfaces was not waiting long enough for the device to complete reset cycle, and prematurely entering recovery mode when there is no real underlying failure. The RPM cannot be automatically installed, as it requires a power cycle as part of the firmware flashing sequence. All power feeds must be manually disconnected and reconnected to cycle it correctly.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.2.md:53]: - **I95-55982 X722 interface MAC being set to `00:00:00:00:00:00` on SSR1300/SSR1400:** Identified an issue the MAC address would change during a power cycle. Another power cycle can restore the MAC to its previous value. An upgrade to the X722 firmware addresses this issue. NOTE: A power cycle is required as part of the firmware flashing sequence. All power feeds must be manually disconnected and reconnected to cycle it correctly.

File: docs/release_notes_128t_6.2.md [hunk 3]
```diff
@@ -24,6 +24,110 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+------
+- **I95-56013 Automatically created Conductor user accounts show as "LDAP":** Resolved an issue with user authentication where accounts were listed as `LDAP` rather than `Remote`.
+------ 
+- **I95-56236 Quick Start config validation failures not being reported:** Made changes to the initialization process such that quick start errors can be reported.
+------
+- **I95-56345 Multiple reboots of the same node of a dual node router causes the stream to stopped:** Resolved an issue where multiple reboots of an HA node did not allow traffic to pass. Now in this scenario an exception is thrown, which allows the session to rebuild once the internode link comes up.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.2.md:59]: - **I95-56345 Multiple reboots of the same node of a dual node router causes multicast stream to stop:** Resolved an issue where multiple reboots of an HA node did not allow traffic to pass. Now in this scenario an exception is thrown, which allows the session to rebuild once the internode link comes up.

File: docs/release_notes_128t_6.2.md [hunk 4]
```diff
@@ -24,6 +24,110 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+------
+- **I95-57017 Application ID failed to block some domains:** Resolved an issue where DPI failed to identify the domain-name from SNI if the `client-hello` is split up into multiple TCP packet segments.
+------
+- **I95-57028 IDP core files consume disk space:** In cases where large amounts of files are stored on disk, the files will be managed/deleted based upon the folder size. 
+------
+- **I95-57071 Changing router context in session debug does not update correctly:** Resolved an issue where a missing dependency was not initiating a re-render of the output using the correct router name. 
+------
+- **I95-57082 Unable to delete a capture-filter that contains a forward slash (/):** This issue has been resolved. 
+------
+- **I95-57099 BFD Session timeout discovered:** Resolved an issue where BFD sessions were deleted prematurely.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.2.md:95]: - **I95-57099 Race condition causing crash in highway process when peer path timers expire:** Resolved an issue with handling BFD timers in multi-threaded environments.

File: docs/release_notes_128t_6.2.md [hunk 5]
```diff
@@ -24,6 +24,110 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+------
+- **I95-57607 Saving TSI as root from the conductor generates oversized file:** Added and enforcement  that when calling `/usr/bin/save-tech-support-info` the `--output` argument always ends in .`zip`.
+------
+- **I95-57692 SSR120/SSR130 Upgrade issues when upgrading from version 6.2.5:** Resolved an issue where if the `128T` process is running and memory usage increases during upgrade, the upgrade will fail due to running out of memory.
+------
+- **I95-57712 DSCP steering issue with outbound traffic:** Resolved an issue with processing reverse pinhole packets when DSCP steering is enabled. 
+------
+- **I95-57853 OTP ISO does not have DHCP server enabled:** Identified and resolved a breaking change with how `logging` is configured. This prevented the DHCP server instance from starting successfully. The default configuration for the DHCP server running in the OTP ISO has been corrected, allowing the server to be started correctly.
+------
+- **WAN-3027 ARV events for peers with hostnames are delayed:**
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.2.md]: incomplete statement.

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

**Review by MichaelBaj — DISMISSED** [review]
Approved, but waiting on confirmation regarding OSPF GR configuration.

**Review by MichaelBaj — APPROVED** [review]

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

**Review by MichaelBaj — APPROVED** [review]

**Review by Bhautik — APPROVED** [review]

**Review by MichaelBaj — APPROVED** [review]

**Pull request merged**